bundles/org.openhab.io.homekit/README.md

```diff
@@ -87,7 +86,7 @@ At the very least, you cannot use repeating (111-11-111) or sequential (123-45-6
 
 Other settings, such as using Fahrenheit temperatures and specifying the interface to advertise the HomeKit bridge are also illustrated in the following sample:
 
-```
+```perl
```

Comment:
Why perl ?

Same question for the other places where you added it.

Reply:
`.cfg` files are not listed in the post that is used as base (for reference: https://github.com/openhab/openhab-docs/issues/999)
As there does not seem to be a preferred one, i choose this as it seems a decent fit. For instance it uses the same comment `#`
But i'm, open for other suggestions.